.github/workflows/smoke-test-prod-podman.yml

```
@@ -0,0 +1,108 @@
+name: Production Smoke Test Podman (via compose.prod.yml)
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  smoke-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Copy environment file
+        run: cp example.prod.env .prod.env
+
+      - name: Add SECRET_KEY to environment file
+        run: |
+          echo -e "\nSECRET_KEY=${{ secrets.SECRET_KEY }}" >> .prod.env
+      - name: Set up Podman
```

Add an extra space here for consistency:

```suggestion
          echo -e "\nSECRET_KEY=${{ secrets.SECRET_KEY }}" >> .prod.env

      - name: Set up Podman
```